index.html

```diff
@@ -7,6 +7,6 @@
     <title>Welcome to Potionify</title>
   </head>
   <body>
-    
+      <h1>Welcome to potionify! Enjoy your stay.</h1>   
```

Comment:
Could you remove the `Enjoy your stay.` string and make the `Welcome to Potionify!` string red?

Reply:
Sure, can do!